client/src/components/QuestionsAndAnswers/components/QuestionAndAnswers.css

```diff
@@ -50,6 +51,72 @@
   transition: all ease 0.3s;
 }
 
-
 /* SearchBar Basic CSS */
+.searchBarWrapper {
+  position: relative;
+  display: flex;
+  min-width: 100px;
+}
+
+.fa {
```

Comment:
@HuijunLu Let's use class names more specific to the widget to avoid conflicts. (`fa`)